### PR TITLE
Credit card payment display

### PR DESCRIPTION
**Pull request by pricem**
This is a small feature directly targeted at #229.  The quantity and total dollar amount of student payments are shown on the dashboard, and in the credit card view module.  Before the program, this consists entirely of credit card payments (it is possible for other payments to be entered by administrators after the program, although that isn't usually done).

### CODE REVIEW

File: esp/esp/program/modules/handlers/creditcardviewer_cybersource.py [hunk 1]
```diff
@@ -59,11 +61,22 @@ def viewpay_cybersource(self, request, tl, one, two, module, extra, prog):
...
         payment_table = []
         
+        #   Fetch detailed information for every student associated with the program
         for student in student_list:
             iac = IndividualAccountingController(prog, student)
             payment_table.append((student, iac.get_transfers(), iac.amount_requested(), iac.amount_due()))
 
-        context = { 'program': prog, 'payment_table': payment_table }
+        #   Also fetch summary information about the payments
+        lt = pac.default_payments_lineitemtype()
```
Comment (jmoldow) [inline esp/esp/program/modules/handlers/creditcardviewer_cybersource.py]: Can you use a more descriptive variable name?

File: esp/esp/program/modules/handlers/jsondatamodule.py [hunk 2]
```diff
@@ -762,6 +763,15 @@ def student_max_count(clslist):
             }
             dictOut["stats"].append({"id": "splashinfo", "data": splashinfo_data})
         
+        #   Add accounting stats
+        lt = ProgramAccountingController(prog).default_payments_lineitemtype()
```
Comment (jmoldow) [inline esp/esp/program/modules/handlers/jsondatamodule.py]: Can you use a more descriptive variable name?

Also, this is the second place where this code is written. Can you add a method on ``ProgramAccountingController`` that returns the tuple ``(num_payments, total_payments)``?

**Comment by jmoldow** [comment]
LGTM, just two minor comments to address.

**Comment by pricem** [comment]
This should be taken care of.

**Comment by benjaminjkraft** [comment]
Merged main to fix tests.

**Comment by benjaminjkraft** [comment]
Looks like @jmoldow's comments are addressed, and I don't see any other problems -- merging.

**Pull request merged**